# Weekly Papers
This project automatically fetches the latest papers from ChemRxiv based on categories.

Only the most recent articles for each category are retained, up to a maximum of 100 papers.

You can click the "Watch" button to receive daily email notifications.

Last update: 2025-05-19

| **Title** | **Date** | **Doi** | **Abstract** | **Categories** | **Suppitems** |
| --- | --- | --- | --- | --- | --- |
| **[DENSE SENSE : A novel approach utilizing an electron density augmented machine learning paradigm to understand a complex odour landscape ](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68225ec1927d1c2e665c424c/original/dense-sense-a-novel-approach-utilizing-an-electron-density-augmented-machine-learning-paradigm-to-understand-a-complex-odour-landscape.pdf)** | 2025-05-15 | 10.26434/chemrxiv-2025-68d6w-v2 | <details><summary>Show</summary><p>Olfaction is a complex process which involves interaction of multiple odour receptors in nasal epithelium to produce the sensation of smell for particular odorant molecules. Elucidating structure-activity-relationships for odorants and their receptors remains difficult since crystallization of the odour receptors extremely difficult. Therefore, ligand-based approaches that leverage machine learning remain the state of the art for predicting odorant properties for molecules, such as the graph neural network approach used by. In this paper we explore how information from Quantum Mechanics (QM) could synergistically improve the results obtained with the graph neural network. Our findings underscore the possibility of this methodology in predicting odour perception directly from QM data, offering a novel approach in the Machine learning space to understand olfaction.</p></details> | Theoretical and Computational Chemistry; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[UMAP-based clustering split for rigorous evaluation of AI models for virtual screening on cancer cell lines](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6821190a50018ac7c5918fac/original/umap-based-clustering-split-for-rigorous-evaluation-of-ai-models-for-virtual-screening-on-cancer-cell-lines.pdf)** | 2025-05-15 | 10.26434/chemrxiv-2024-f1v2v-v2 | <details><summary>Show</summary><p>Virtual Screening (VS) of large compound libraries using Artificial Intelligence (AI) models is a highly effective approach for early drug discovery. Data splitting is crucial for benchmarking the performance of such AI models. Traditional ran-dom data splits often result in structurally similar molecules in both training and test sets, which conflict with the reality of VS libraries that typically contain structurally diverse compounds. To tackle this challenge, scaffold split, which groups molecules by shared core structure, and Butina clustering, which clusters molecules by chemotypes, have long been used. However, we show that these methods still introduce high similarities between training and test sets, leading to overestimated model performance. Our study examined four representative AI models across 60 NCI-60 datasets, each comprising approximately 33,000 to 54,000 molecules tested on different cancer cell lines. Each dataset was split in four ways: random, scaffold, Butina clustering and the more realistic Uniform Manifold Approximation and Projection (UMAP) clustering. Using Linear Re-gression, Random Forest, Transformer-CNN, and GEM, we trained a total of 8,400 models and evaluated under four splitting methods. These comprehensive results show that UMAP split provides more challenging and realistic bench-marks for model evaluation, followed by Butina splits, then scaffold splits and closely after random splits. Consequently, we recommend using UMAP splits instead of overly optimistic Butina splits and especially scaffold splits for mo-lecular property prediction, including VS. Lastly, we illustrate how misaligned ROC AUC is with VS goals, despite its common use. The code and datasets for reproducibility are available at https://github.com/Rong830/UMAP_split_for_VS and archived in https://zenodo.org/records/14736486</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[Computational Studies Reveal Insights on a Light-Harvesting System with Phycocyanobilin](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681fadbd927d1c2e66f7e963/original/computational-studies-reveal-insights-on-a-light-harvesting-system-with-phycocyanobilin.pdf)** | 2025-05-14 | 10.26434/chemrxiv-2025-6jzqt | <details><summary>Show</summary><p>Biological light-harvesting systems comprise complex protein-pigment assemblies that exhibit vast structural and functional diversity, underscoring the design principles of functional bio-inspired materials applicable to sustainable energy-harvesting technologies.  Thus, this investigation aimed to decode the structural complexity and assembly of protein-pigment complexes containing phycocyanobilin, an open-chain tetrapyrrole chromophore.  This study employed the synergistic application of computational tools, including artificial intelligence-based methods and density functional theory (DFT) calculations, to deconstruct the structural framework of the photosynthetic macromolecular system found in autotrophic organisms.  The results highlight structural similarities in the protein-based scaffolds and contrasting spectroscopic fingerprints of the pigments in “protein-like” environments, outlining the key features of their structural organization and underpinning the connection between structural biology and systems chemistry. </p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Materials Science; Biophysics; Computational Chemistry and Modeling; Artificial Intelligence | <details><summary>Show</summary><p>[]</p></details> |
| **[Artificial Intelligence Based in silico Clinical Trials for Vaccines](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681758c5927d1c2e66f17512/original/artificial-intelligence-based-in-silico-clinical-trials-for-vaccines.pdf)** | 2025-05-14 | 10.26434/chemrxiv-2025-3nh8w | <details><summary>Show</summary><p>This paper explores the transformative potential of Artificial Intelligence (AI) in enabling in silico clinical trials (ISCTs) for vaccine development. ISCTs, which simulate clinical trials using virtual patient cohorts, offer a cost-effective, ethical, and efficient alternative to traditional methods. AI significantly enhances ISCTs by automating trial design, optimizing patient selection, and simulating immune responses with unprecedented accuracy and scale. Applications include antigen discovery, vaccine formulation, efficacy and safety prediction, and personalized immunization strategies. Through a review of current research, tools, and industry practices, the paper illustrates how AI-driven ISCTs can reduce time-to-market, lower costs, improve trial diversity, and meet regulatory expectations. Despite challenges such as model validation, data bias, and regulatory uncertainties, AI-integrated ISCTs are positioned to revolutionize vaccine R&D. The paper concludes by highlighting future trends, including the integration of AI with emerging technologies, development of universal and personalized vaccines, and evolving regulatory frameworks to support these innovations.</p></details> | Theoretical and Computational Chemistry; Biological and Medicinal Chemistry; Bioinformatics and Computational Biology; Drug Discovery and Drug Delivery Systems; Artificial Intelligence | <details><summary>Show</summary><p>[]</p></details> |
| **[Aitomia: Your Intelligent Assistant for AI-Driven Atomistic and Quantum Chemical Simulations](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681c98d0927d1c2e668e7854/original/aitomia-your-intelligent-assistant-for-ai-driven-atomistic-and-quantum-chemical-simulations.pdf)** | 2025-05-13 | 10.26434/chemrxiv-2025-gnf13 | <details><summary>Show</summary><p>We have developed Aitomia – a platform powered by AI to assist in performing AI-driven atomistic and quantum chemical (QC) simulations. This intelligent assistant platform is equipped with chatbots and AI agents to help experts and guide non-experts in setting up and running the atomistic simulations, monitoring their computation status, analyzing the simulation results, and summarizing them for the user in text and graphical forms. We achieve these goals by exploiting fine-tuned open-source large language models (LLMs), rule-based agents, and a retrieval-augmented generation (RAG) system. Aitomia leverages the versatility of our MLatom ecosystem for AI-enhanced computational chemistry. This intelligent assistant is going to be integrated into the Aitomistic Hub and XACS online computing services, with some functionality already publicly available as described at http://mlatom.com/aitomia. Aitomia is expected to lower the barrier to performing atomistic simulations, accelerating research and development in the relevant fields.</p></details> | Theoretical and Computational Chemistry; Artificial Intelligence | <details><summary>Show</summary><p>[]</p></details> |
| **[Predicting Reaction Feasibility and Selectivity of Aromatic C–H Thianthrenation with a QM-ML Hybrid Approach](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6823546950018ac7c5f0504d/original/predicting-reaction-feasibility-and-selectivity-of-aromatic-c-h-thianthrenation-with-a-qm-ml-hybrid-approach.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2025-pvpnv | <details><summary>Show</summary><p>The direct thianthrenation of aromatic C–H bonds is a valuable late-stage functionalization strategy that can assist, for example, the development of new drugs. We herein present a predictive computational model for this reaction, denoted PATTCH, which is based on semi-empirical quantum mechanics and machine learning. It classifies each aromatic C–H unit either as reactive or not with an accuracy of above 90%. It can address both the site-selectivity and reaction feasibility question associated with the thianthrenation protocol. First, this was achieved by selecting carefully engineered features, which take into account the electronic and steric influence on the site-selectivity. Second, parallel experimentation was used to supplement the available literature data with 54 new negative reactions (unsuccessful thianthrenation), which we show was instrumental for developing the PATTCH tool. Ultimately, we successfully applied the model to a challenging test set encompassing the differentiation between carbocycle vs. heterocycle functionalization, the identification of substrates that were reported to result in a mixture of isomeric products, and to molecules that could not be thianthrenated. The computational predictions were experimentally validated. The PATTCH tool can be obtained free of charge from https://github.com/MolecularAI/thianthrenation_prediction.</p></details> | Theoretical and Computational Chemistry; Organic Chemistry; Computational Chemistry and Modeling; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/682368d2927d1c2e66902ee9/original/supporting-information.pdf"}]</p></details> |
| **[Machine Learning Models for Predicting Electronic Coupling in TEMPO/TEMPO+ Systems](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6824ac7650018ac7c5247fba/original/machine-learning-models-for-predicting-electronic-coupling-in-tempo-tempo-systems.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2025-rk2bd | <details><summary>Show</summary><p>Organic Radical Batteries (ORBs) based on the TEMPO (2,2,6,6-tetramethylpiperidin-1-yl oxyl) radical have drawn significant attention owing to their unique redox properties. A key factor influencing ORB's redox properties, i.e., the kinetic of the electron transfer between the TEMPO-TEMPO+ pairs is the communication between the underlying redox-active states as given by the electronic coupling. However, due to the complex structure, predicting accurate electronic couplings for these pairs is computationally expensive and challenging.  In this study, we introduce a machine learning (ML) workflow to predict the electronic coupling for TEMPO-TEMPO+ pairs simply from their specific geometric orientations. For the ML models, a dataset was generated through time-dependent density functional theory calculations coupled with the Generalized Mulliken Hush method to assess energies, (transition-)dipole moment and couplings for specific TEMPO-TEMPO+ configurations obtained from classical molecular dynamics simulations that mimic a realistic electrolyte environment. Our results demonstrate that, among the three ML models—linear regression, kernel ridge regression (KRR), and random forest—the KRR model, with its kernel-based approach, most effectively handles the correlated orientation-based descriptors. Moreover, our SHapley Additive exPlanations (SHAP)-based feature importance analysis indicates that multiple orientation factors jointly influence electronic the coupling, rather than any single distance or angle dominating, with each parameter’s impact strongly contingent on the values of the others which is in agreement with previous studies computational by the consortium.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational; Machine Learning | <details><summary>Show</summary><p>[]</p></details> |
| **[Understanding Equilibrium Size Distribution of Metal Single Atoms, Clusters, and Particles from Thermodynamics](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68229d49e561f77ed479f4fe/original/understanding-equilibrium-size-distribution-of-metal-single-atoms-clusters-and-particles-from-thermodynamics.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2025-vx5jp | <details><summary>Show</summary><p>Dynamic size evolution of supported metal single atoms, clusters and nanoparticles widely observed through in-situ characterization techniques plays a crucial role on the catalytic activity and stability. However, the underlying atomic-level mechanism remains elusive. To gain insights into this dynamic evolution phenomenon, we have derived a universal equilibrium size distribution equation,  through a variational method from a thermodynamic perspective (see below). This equation holds under both canonical and grand canonical ensemble, and can be used to predict how particle size distributions change with the environment and the total metal loading ratio, thereby revealing the trends of size evolution without a priori knowledge on kinetics. By integrating machine learning force filed (MLFF) with Monte Carlo (MC) and molecular dynamics (MD) simulations, the size evolution is modelled and the results are consistent with the physical insights derived from our equilibrium size distribution equation.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Nanoscience; Computational Chemistry and Modeling; Machine Learning; Thermodynamics (Physical Chem.) | <details><summary>Show</summary><p>[{"title": "Supplementary Materials for Understanding Equilibrium Size Distribution of Metal Single Atoms, Clusters, and Particles from Thermodynamics", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6822a0b5e561f77ed47a6a32/original/supplementary-materials-for-understanding-equilibrium-size-distribution-of-metal-single-atoms-clusters-and-particles-from-thermodynamics.pdf"}]</p></details> |
| **[Mining Chemical Space with Generative Models
for Battery Materials](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68093169927d1c2e666f1055/original/mining-chemical-space-with-generative-models-for-battery-materials.pdf)** | 2025-05-15 | 10.26434/chemrxiv-2025-q48jr | <details><summary>Show</summary><p>Generative models represent a powerful new paradigm for accelerating the discovery of novel materials across vast chemical space. To evaluate the viability of deploying generalized crystal generative models for application specific discovery tasks, here we choose Li-ion battery(LIB) materials as a case study. The pretrained MatterGen model is used to generate diverse crystalline structures, conditioned for stability and tested for uniqueness and novelty for promising Li-containing compositions. We performed unsupervised clustering analysis using atomic neighborhood fingerprints to compare the distribution of generated structures against the training dataset and materials project(MP) data in the chemical space. Our multi-tired workflow for LIB materials combines universal crystal generative model with foundational machine learning potential to identify most promising stable (close to convex hull with respect to MP data) candidates for final Density Functional Theory (DFT) based stability calculations. Open Circuit Voltage (OCV) and specific capacity calculations on selected stable materials highlighted their potential as LIB materials. Among 91 identified Li-containing stable (0.03 eV/atom above MP convex hull) materials, we identified three novel cathode materials useful for LIB considering  average OCV, OCV at highest and lowest state of charge, and the specific capacity. </p></details> | Theoretical and Computational Chemistry; Machine Learning; Artificial Intelligence; Materials Chemistry | <details><summary>Show</summary><p>[{"title": "supporting data", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68225ef5927d1c2e665c7bbe/original/supporting-data.docx"}]</p></details> |
| **[Q-DFTNet: A Chemistry-Informed Neural Network Framework for Predicting Molecular Dipole Moments via DFT-Driven QM9 Data](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681e4e6d50018ac7c533a0cc/original/q-dft-net-a-chemistry-informed-neural-network-framework-for-predicting-molecular-dipole-moments-via-dft-driven-qm9-data.pdf)** | 2025-05-14 | 10.26434/chemrxiv-2025-8j3km | <details><summary>Show</summary><p>Predicting molecular dipole moments is essential for quantum chemistry and materials science applications. In this study, we introduce Q-DFTNet: a Chemistry-Informed Neural Network framework designed to systematically benchmark and interpret graph neural networks (GNNs) for molecular dipole prediction. Seven GNN architectures: GCN, GIN, GraphConv, GATNet, GATConv, SAGEConv, and GIN+EdgeConv, were trained on the QM9 dataset for 100 epochs and evaluated using mean squared error (MSE), mean absolute error (MAE), coefficient of determination (R$^2$), and clustering validity. GraphConv attained the best predictive performance with a test MSE of 0.8514, MAE of 0.6407, and R$^2$ of 0.621, outperforming other models with only 16.5k parameters. Attention-based models (GATNet, GATConv) exhibited faster convergence but lower accuracy, with MSEs exceeding 1.09 and R$^2$ scores below 0.52. Latent spaces were visualized via t-SNE, PCA, and UMAP projections, revealing structural encoding differences across models. Clustering quality, assessed using Silhouette Score (0.4658 for GraphConv), Davies-Bouldin Index (0.7476 for GCN), and Calinski-Harabasz Score (2411.73 for GraphConv), demonstrated how Q-DFTNet captures chemically coherent groupings. A k-means analysis of 1000 molecules in latent space revealed that GIN+EdgeConv formed the most interpretable clusters, with dipole means ranging from 2.407 to 2.778 Debye. Q-DFTNet provides a unified framework for prediction, representation learning, and molecular interpretability, offering valuable guidance for neural architecture selection in quantum machine learning pipelines.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Machine Learning; Quantum Computing | <details><summary>Show</summary><p>[]</p></details> |
| **[tmQMg* Dataset: Excited State Properties of 74k Transition Metal Complexes](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681dadb6e561f77ed4c7421c/original/tm-q-mg-dataset-excited-state-properties-of-74k-transition-metal-complexes.pdf)** | 2025-05-13 | 10.26434/chemrxiv-2025-pdd8p | <details><summary>Show</summary><p>The application of machine learning approaches to meaningful problems in chemistry and materials science is still challenged by the limited availability of  data. In order to close this gap, we report the tmQMg* dataset, which provides excited state properties for 74k mononuclear transition metal complexes extracted from the Cambridge Structural Database. All properties were computed at the TD-DFT wB97xd/def2SVP level of theory. The strongest electron excitations in the ultraviolet, visible, and near-infrared ranges are included, together with the wavelengths and intensities of the first 30 excited states. Further, natural transition orbitals were computed for the strongest excitation in the visible range to determine the nature of the associated charge transfer. By computing the TD-DFT spectra in both gas phase and acetone, we quantified  solvatochromic effects, which are also provided with the dataset, in terms of both wavelength shifts and intensity changes. The tmQMg* dataset will enable the development of discriminative and generative AI models with respect to  absorption spectra, charge transfer character, and solvatochromism, enabling novel advances in the field of transition metal photochemistry.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[Computational Exploration of Ni-P Bulk Metallic Glasses via Machine Learning Potentials](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681d0d1e50018ac7c509812c/original/computational-exploration-of-ni-p-bulk-metallic-glasses-via-machine-learning-potentials.pdf)** | 2025-05-13 | 10.26434/chemrxiv-2025-grvb3 | <details><summary>Show</summary><p>Bulk metallic glasses (BMGs) are a unique class of materials characterized by their disordered atomic structure, which imparts exceptional mechanical strength, corrosion resistance, and catalytic activity. In this study, we develop a machine learning interatomic potential (MLIP) for Ni-P-based BMGs using an artificial neural network approach. A comprehensive dataset spanning a broad range of temperatures, pressures, and compositions was generated through atomistic simulations. The MLIP was validated against density functional theory calculations, demonstrating high accuracy in predicting energy and forces. The developed potential allowed us to investigate the local atomic structure and dynamic behavior of Ni-P BMGs across different Ni-P ratios and temperature regimes. Our findings reveal the role of short- and medium-range ordering in the structural stability of these BMGs, highlighting the influence of phosphorus concentration on glass-forming ability. Additionally, we explored the upper compositional limit of P that can sustain the amorphous nature of Ni-P BMGs. The results provide critical insights into the atomic arrangement for the formation and stability of BMGs.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Machine Learning; Physical and Chemical Properties; Structure; Materials Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[Librarian of Alexandria: An Extensible LLM-based Chemical Data Extraction Pipeline](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68168c3c50018ac7c54cff43/original/librarian-of-alexandria-an-extensible-llm-based-chemical-data-extraction-pipeline.pdf)** | 2025-05-13 | 10.26434/chemrxiv-2025-fb8hj | <details><summary>Show</summary><p>Dataset creation is a critical component of predictive machine learning technology. In the field of chemistry, large experimental datasets are scarce, especially for niche chemical properties and topics, and their creation is both cumbersome and time-consuming. Here we present Librarian of Alexandria (LoA), an open-source automatic tool for large dataset generation via direct extraction by large- language models (LLM) from scientific literature. LoA is available on GitHub along with example inputs, outputs and a Colab-friendly Jupyter Notebook (see Supplementary Information). LoA checks the relevance of a research paper and performs data extraction using two separate LLMs which may be independently and modularly specified by the end-user. LoA can be easily updated via simplified user incorporation of the latest available LLMs. We also compared several LLMs for both relevance and extraction functions, and automated the collection of research papers for several popular chemical journals which provide open access. LoA provides a means by which enormous chemical datasets can be created with minimal effort and ∼80% accuracy for the purposes of training predictive machine learning models.</p></details> | Theoretical and Computational Chemistry; Machine Learning; Artificial Intelligence; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[{"title": "Supplemental Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681bbb83e561f77ed47c0b41/original/supplemental-information.pdf"}]</p></details> |
| **[How to generalize machine learning models to both canonical and non-canonical peptides](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681ba2a650018ac7c5d1d4b2/original/how-to-generalize-machine-learning-models-to-both-canonical-and-non-canonical-peptides.pdf)** | 2025-05-12 | 10.26434/chemrxiv-2025-ggp8n-v2 | <details><summary>Show</summary><p>Bioactive peptides are an important class of natural products with great functional diversity. Chemical modifications can improve their pharmacology, yet their structural diversity presents unique challenges for computational modeling. Furthermore, data for canonical peptides (non-modified) is more abundant than for non-canonical (chemically modified). We set out to identify the feasibility of generalizing from canonical to non-canonical datasets. To do this, we first considered two critical aspects of the modeling problem, namely, choice of similarity function for guiding dataset partitioning and choice of molecular representation. Similarity-based dataset partitioning is an evaluation technique that divides the dataset into train and test subsets, such that the molecules in the test set are different from those used to fit the model. We demonstrate, across four peptide function prediction tasks, that chemical fingerprint-based similarity measures outperform traditional sequence alignment-based metrics for partitioning canonical peptide datasets, challenging standard practices. Similarly, we have found that chemical fingerprints are the best option for building canonical-to-canonical and non-canonical-to-non-canonical predictive models. for the more challenging canonical-to-non-canonical and non-canonical-to-canonical extrapolation scenarios. Finally, we discovered that by enriching the canonical datasets with non-canonical peptides, we are able to build robust joint models that generalise adequately to both canonical and non-canonical data. All code and data necessary for reproducing the experiments are available in Github (https://github.com/IBM/PeptideGeneralizationBenchmarks)</p></details> | Theoretical and Computational Chemistry; Machine Learning; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[Protein-Based QM-CGMM](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681e427250018ac7c53263c1/original/protein-based-qm-cgmm.pdf)** | 2025-05-14 | 10.26434/chemrxiv-2025-xp7w1-v3 | <details><summary>Show</summary><p>We present an approach to Coarse Grain (CG) the Molecular Mechanics (MM) region in a thermodynamically consistent approach when the region is a protein backbone. This approach is achieved via using 4 beads on each amino acid residue to replicate the electrostatic field that it generates, followed by connecting them via a heterogeneous Elastic Network Model (heteroENM) force-field. We subsequently treat the boundary between the QM and CGMM regions using appropriate Lennard-Jones parameters. We apply this procedure to the classic Chorismate Mutase system, where the enzyme backbone in the MM region catalyses the Claisen rearrangement in the QM region using Transition-Tempered metadynamics (TTMetaD) to obtain a converged Potential of Mean Force (PMF). The CGMM reveals a free energy barrier similar to the all-atom QM/MM counterpart. This work demonstrates the potential of utilizing a CGMM forcefield to capture all-atom thermodynamic properties at a reduced cost.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supporting Information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681e427250018ac7c53263c9/original/supporting-information.pdf"}]</p></details> |
| **[Vibronic Spectrum of Pyrazine: New Insights from Multi-state-multi-mode Simulations Parameterized with Equation-of-Motion Coupled-Cluster Methods](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6823bee5e561f77ed4ac8c5d/original/vibronic-spectrum-of-pyrazine-new-insights-from-multi-state-multi-mode-simulations-parameterized-with-equation-of-motion-coupled-cluster-methods.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2025-tnfsl | <details><summary>Show</summary><p>This study reports simulations of the lowest band in the electronic absorption spectrum of pyrazine carried out using multi-state-multimode vibronic Hamiltonian parameterized using equation-of-motion coupled-cluster methods. The simulations explain the main spectral features and show how peaks of vibronic nature appear. The most complete vibronic model includes four electronic states and six vibrational modes. The simulations reveal that non-adiabatic coupling with bright states lo- cated as high as 3 eV above the studied state can lead to discernable features in the absorption spectrum. This study demonstrates the power of fully ab initio treat- ments of electronic and vibrational structure and their utility in understanding the mechanisms leading to complex molecular spectra.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[]</p></details> |
| **[Benchmarking of Vibrational Exciton Models Against Quantum-Chemical Localized-Mode Calculations](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/682366dfe561f77ed49e1b9a/original/benchmarking-of-vibrational-exciton-models-against-quantum-chemical-localized-mode-calculations.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2024-s2jgl-v2 | <details><summary>Show</summary><p>Vibrational exciton models are widely used for the simulation of biomolecular vibrational spectra, in particular of two-dimensional infrared (2D-IR) spectra. The parameters entering such models, specifically local-mode frequencies and coupling constants, are provided by vibrational maps, which have been parametrized against computational data for small molecules as well as experimental data. Here, we put forward a novel approach for assessing the quality of these vibrational maps against quantum-chemical reference data. For a test set consisting of molecular dynamics snapshots of polypeptides and small proteins, covering different secondary structure motifs, we we performed full quantum-chemical calculations of harmonic vibrational frequencies and normal modes, and applied a localization of normal modes to obtain localized-mode frequencies and coupling constants. These can be directly compared to those predicted by vibrational maps. We find that while there is a good correlation for the coupling constants and for local-mode frequencies of isolated polypeptides, there is hardly any correlation for the local-mode frequencies of solvated polypeptides. This striking finding calls into question the accuracy of the electrostatic maps that are used to model the effect of the solvent molecules on local-mode frequencies.</p></details> | Theoretical and Computational Chemistry; Physical Chemistry; Theory - Computational; Spectroscopy (Physical Chem.) | <details><summary>Show</summary><p>[{"title": "Supporting information", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/68236781927d1c2e66900ce7/original/supporting-information.pdf"}]</p></details> |
| **[Improving Operator Splitting and Effective
Reaction Probability for A Reactive-Step Based
Molecular Dynamics Workflow](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6824a139e561f77ed4c96541/original/improving-operator-splitting-and-effective-reaction-probability-for-a-reactive-step-based-molecular-dynamics-workflow.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2025-xk26c | <details><summary>Show</summary><p>	Classical molecular dynamics (MD) simulation is the most computationally efficient way to model large molecular systems atomistically for extended periods; however, due to fixed forcefield parameters, incorporating on-the-fly quantum reactions is not straightforward. Reactive Step-Based Molecular Dynamics (RSMD) is a simple approach that incorporates quantum reactions by periodically halting the MD simulation and allowing the possibility for reactions at each halt, based on a Poisson-type reaction probability. However, this simple approach cannot capture the simultaneous involvement of diffusion and reaction processes, and because the reaction probability does not include the influence of the diffusion step, errors can be introduced, especially when the diffusion process is not significantly slower than the reaction process. In this work, the efficiency of the RSMD model is increased by reducing these errors and by addressing the influence of the diffusion process on the reaction probability. To improve these errors, we modify the RSMD mathematical framework by replacing the Trotter splitting employed in previous works with the Strang splitting scheme. To implement these Strang schemes in MD simulations and to scrutinize their validity, we introduce mathematical models involving three and four states, which correspond to two common reaction scenarios---association-dissociation reactions and homogeneous charge transfer reactions, respectively---in the presence of diffusion processes. Using these mathematical models, effective reaction probability functions are derived for various diffusion limits, for example, when diffusion processes are extremely fast or extremely slow compared to the reaction processes. All the derived reaction probability functions, in combination with appropriate Strang schemes, are validated for various diffusion regimes with respect to the reaction timescale.	 </p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[]</p></details> |
| **[High-Performance, High-Angular-Momentum J Engine on Graphics Processing Units](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6822bd44927d1c2e66706835/original/high-performance-high-angular-momentum-j-engine-on-graphics-processing-units.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2025-2lntb | <details><summary>Show</summary><p>Efficient evaluation of electron repulsion integrals (ERIs) involving high-angular-momentum Gaussian basis functions is computationally challenging on graphical processing units (GPUs), as traditional recurrence-based integral algorithms generate numerous intermediates, causing significant register pressure and memory bottlenecks.  In this Article, we present a high-performance, high-angular-momentum Coulomb-matrix J engine specifically optimized for GPU execution. Our approach introduces a novel GPU-optimized McMurchie-Davidson recurrence algorithm combined with a tailored integral batching scheme, designed specifically to jointly minimize intermediate storage requirements and redundant computation. By strategically partitioning high-angular-momentum ERIs classes into several carefully selected sub-batches, our approach transitions the associated integral evaluation kernels from memory-bound to compute-bound regimes, significantly enhancing computational throughput and reducing time to solution.   Implemented in the Extreme-scale Electronic Structure System (EXESS), our algorithm achieves individual kernel speedups of up to 9x and improves overall J-matrix formation performance by up to 64% across a variety of increasing-size chemical systems, including polyglycine chains, water clusters, and boron nitride crystals, when using the cc-pVQZ quadruple-zeta basis set.</p></details> | Theoretical and Computational Chemistry; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supplementary Material: A GPU accelerated J matrix engine for high angular momentum", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6822be74e561f77ed47eee1c/original/supplementary-material-a-gpu-accelerated-j-matrix-engine-for-high-angular-momentum.pdf"}]</p></details> |
| **[Prediction of Potential Drug-Resistant ALK Kinase Mutations against Fourth-Generation Inhibitors NVL-655 and TPX-0131](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6822ab1c50018ac7c5d5d17f/original/prediction-of-potential-drug-resistant-alk-kinase-mutations-against-fourth-generation-inhibitors-nvl-655-and-tpx-0131.pdf)** | 2025-05-16 | 10.26434/chemrxiv-2025-cdhlh | <details><summary>Show</summary><p>Anaplastic lymphoma kinase (ALK) rearrangements have been identified as oncogenic drivers of multiple malignancies. Although at least three generations of ALK inhibitors have been developed, the occurrence of drug-resistant mutations remains a major barrier to drug efficacy. In this study, we develop a multiscale computational pipeline that combines hotspot residue identification, saturation mutagenesis screening, and alchemical free energy calculations to predict the impact of point mutations on the binding affinity of fourth-generation ALK inhibitors NVL-655 and TPX-0131. The hotspot residue analysis reveals that L1122, V1130, V1180, L1196, L1198, M1199, D1203, and L1256 contribute the most to ligand binding. A total of 9 and 13 mutations are identified as drug-resistant candidates with significantly decreased binding free energy of over 3 kcal/mol for NVL-655 and TPX-0131, respectively. Specifically, V1180W, M1199W, and L1256S are the common mutations with decreased binding free energy concerning both inhibitors. These findings highlight important residues and mutations that may impact the clinical efficacy of NVL-655 and TPX-0131, and this pipeline provides an efficient and accurate framework to predict drug-resistant mutations and facilitate the rational design of next-generation ALK inhibitors.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supporting information for: Prediction of Potential Drug-Resistant ALK Kinase Mutations against Fourth-Generation Inhibitors NVL-655 and TPX-0131", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6822b2f2e561f77ed47d4d98/original/supporting-information-for-prediction-of-potential-drug-resistant-alk-kinase-mutations-against-fourth-generation-inhibitors-nvl-655-and-tpx-0131.docx"}, {"title": "Figures", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/6822b31ee561f77ed47d51fd/original/figures.zip"}]</p></details> |
| **[Weighted Ensemble Simulation: Advances in methods, software, and applications](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681bddffe561f77ed483d613/original/weighted-ensemble-simulation-advances-in-methods-software-and-applications.pdf)** | 2025-05-13 | 10.26434/chemrxiv-2025-jtppp | <details><summary>Show</summary><p>For more than two decades, weighted ensemble (WE) path sampling strategies have enabled the simulation of pathways for rare events—or barrier-crossing processes—with significantly less computing cost than conventional simulations, all while preserving rigorous kinetics. This review highlights recent advances in WE methods and software, including tools for mechanistic analysis of path ensembles and efficient estimation of rates. We showcase successful weighted ensemble applications across a wide range of condensed-phase processes, such as hybrid quantum mechanics/molecular mechanics (QM/MM) simulations of microsecond-timescale chemical reactions, and atomistic simulations of slower processes on the millisecond to seconds timescale. These applications span drug membrane permeation, ligand unbinding, and the large-scale opening of the SARS-CoV-2 spike protein. We also discuss the current limitations and key challenges facing WE strategies, which have yet to reach their full potential. </p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational; Chemoinformatics - Computational Chemistry | <details><summary>Show</summary><p>[]</p></details> |
| **[Polynomial Scaling Localized Active Space  Unitary Selective Coupled Cluster Singles and Doubles](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681ccc0b50018ac7c5fecff4/original/polynomial-scaling-localized-active-space-unitary-selective-coupled-cluster-singles-and-doubles.pdf)** | 2025-05-12 | 10.26434/chemrxiv-2025-1n3d4-v2 | <details><summary>Show</summary><p>We present a polynomial-scaling algorithm for the localized active space unitary selective coupled cluster singles and doubles (LAS-USCCSD) method. In this approach, cluster excitations are selected based on a threshold $\mathnormal{\epsilon}$ determined by the absolute gradients of the LAS-UCCSD energy with respect to cluster amplitudes. Using the generalized Wick's theorem for multireference wave functions, we derive the gradient expression as a polynomial function of 1-, 2-, and 3-body reduced density matrices and 1- and 2-electron integrals, valid for any multireference wave function.   The resulting gradient implementation exhibits a memory scaling of $\mathcal{O}(N^6)$.    The variational quantum eigensolver is used to optimize the selected cluster excitations on a quantum simulator.   By plotting the energy error, defined as the difference between the LAS-USCCSD and corresponding CASCI energies, against the inverse cluster amplitude selection threshold ($\mathnormal{\epsilon}^{-1}$) for polyene chains containing 2 to 5 $\pi$-bond units, we establish a relationship between the energy error and the threshold.   To further validate the accuracy of LAS-USCCSD, we computed the cis–trans isomerization energy of stilbene (a 20-qubit system) and the magnetic coupling constant of the tris-hydroxo-bridged chromium dimer \ce{[Cr_2(OH)_3(NH_3)_6]^{3+}} (evaluated as both 12- and 20-qubit systems) using the Qiskit-Qulacs simulator. Assessing such examples is important to determine the practical feasibility of quantum simulations for chemically realistic systems. Toward this goal, with this LAS-USCCSD algorithm we estimated the quantum resources required for simulating an active space of (30e,22o) in \ce{[Cr_2(OH)_3(NH_3)_6]^{3+}}, a size that remains beyond the reach of current quantum simulators for accurate treatment.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational; Quantum Computing | <details><summary>Show</summary><p>[]</p></details> |
| **[Velocity Rescaling in Surface Hopping Based on Atomic Contributions to Electronic Transitions](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681b381e50018ac7c5c40aab/original/velocity-rescaling-in-surface-hopping-based-on-atomic-contributions-to-electronic-transitions.pdf)** | 2025-05-12 | 10.26434/chemrxiv-2025-gc95h | <details><summary>Show</summary><p>Surface hopping is a widely used method for simulating nonadiabatic dynamics, in which nuclear motion follows classical trajectories and electronic transitions occur stochastically. To ensure energy conservation during these transitions, atomic velocities must be adjusted. Traditional velocity rescaling methods either apply a uniform adjustment to atomic velocities, which can lead to size consistency issues, or rely on nonadiabatic coupling vectors, which are computationally expensive and may not always be available. Here, we introduce two novel velocity rescaling methods that incorporate atomic contributions to electronic transitions, derived from the one-electron transition density matrix or the density difference between states for a given transition. The first method, excitation weighted velocity rescaling, redistributes kinetic energy among atoms proportionally to their contributions to the electronic transition. This is achieved through a weighted scaling factor, computed from the population analysis of the one-electron transition density matrix or the density difference of the two states involved in the transition. The second method, excitation-thresholded velocity rescaling, adjusts the velocities only of atoms whose contributions exceed a predefined threshold, preventing unnecessary energy redistribution to atoms with minimal involvement in the excitation. We validate these approaches through excited-state dynamics simulations of fulvene and 1H-1,2,3 triazole. Our results show that excitation-weighted velocity rescaling closely reproduces the adjustments based on nonadiabatic coupling vectors for both fulvene and 1H-1,2,3-triazole.</p></details> | Theoretical and Computational Chemistry; Computational Chemistry and Modeling; Theory - Computational | <details><summary>Show</summary><p>[{"title": "Supplementary Information for: \"Velocity Rescaling in Surface Hopping Based on Atomic Contributions to Electronic Transitions\"", "assetLink": "https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/681b3a3850018ac7c5c42d04/original/supplementary-information-for-velocity-rescaling-in-surface-hopping-based-on-atomic-contributions-to-electronic-transitions.pdf"}]</p></details> |

